main.py

```diff
@@ -73,7 +73,7 @@ def should_nag(message):
 
 def _is_video_tweet(embed):
     print(embed)
-    return embed.video and "twitter.com" in embed.url
+    return embed.video and (True if re.search("(x|twitter)\.com", embed.url) else False)
```

Comment:
Similar to L65, should use raw strings for regex by convention. Also, for what it's worth, two simple string searches (as in, ``"twitter.com" in embed.url or "x.com" in embed.url``) is still about 5-10x faster than a regex search (not that this is terribly performance-critical either way).